samples/network/README.md

```diff
@@ -0,0 +1,63 @@
+# How to use Berkeley/Posix Socket APIs in WebAssembly
```

Comment:
Will us add a document under `<wamr_root>/doc/`, e.g. doc/socket_api.md?
And add introduction and link under the README `Key features`?